merlin/merlin_test.go

```diff
@@ -37,15 +37,20 @@ var _ = Describe("merlin", func() {
 
```

Comment:
It seems like each test has checks for both http and https servers. 
I wonder if using the [ginkgo shared examples](http://onsi.github.io/ginkgo/#pattern-1-extract-a-function-that-defines-the-shared-its) would help? you could write the shared scenarios, and then have two contexts, one http and one https, and then just apply those checks. 
Give it a look and see if it would make sense.

Reply:
I'm not sure I follow - there is only a single assertion on the merlin client mock.

You mean stubbing the mocks? I'm hesitant to put that in a method because it obscures the test meaning.

Reply:
nope, it is something similar as the `shared_examples` for the rspecs, where you define the expected cases, and then apply them to different targets. 
But i just realized there are only 3 tests like that, so not really worth it. Somehow I though there were many more.